.goreleaser.yml

```diff
@@ -47,8 +47,21 @@ signs:
       - "${signature}"
       - "--detach-sign"
       - "${artifact}"
+changelog:
+  skip: false
+  use: github
+  groups:
+    - title: Features
+      order: 0
+      regexp: "^[Ff]eat[: ].*$"
```

Comment:
@yaronya 
1. This should regex commits.
2. I think it's a good idea. Would we want to validate the commit and branch names? - I can do it as a follow-up PR (this ticket stays open since there is more work to be done here anyway).
3. I was hoping it creates a CHANGELOG.md file - but I really don't know how to test it. May need to push... release a DEV tag... and see what happens.

BTW: this one is for releases https://goreleaser.com/customization/release/

Reply:
1. So since we're squashing commits we're good with that I suppose
2. I'd validate what matters, which seems to be the commit (that derives from the PR name? 🤔 )
3. 🆗 

> BTW: this one is for releases [goreleaser.com/customization/release](https://goreleaser.com/customization/release/)

Maybe we can use that one instead of or alongside `CHANGELOG.md`? WDYT?

Reply:
Agreed! 
Alongside. (This is why I'm keeping the issue open - more work to get done)